src/app/service/recap/ServiceRecap.jsx

```diff
@@ -8,6 +8,12 @@ import SectionHeader from '../../../components/sectionHeader';
 import NavBar from '../../NavBar';
 import ListItem from './ListItem';
 
+const iconNames = {
```

Comment:
Could this be moved to a util or something, so it can be shared between ServiceCategories and this?